content/overview/getting-started/_index.en.md

```diff
@@ -5,50 +5,149 @@ weight: 2
 draft: false
 ---
 
-### Global site parameters
+In this guide, we'll be covering the following starting actions for wasmCloud development:
+1. Launching a wasmCloud Host
+1. Starting a wasmCloud Actor
+1. Starting a wasmCloud Capability Provider
+1. Linking an Actor to a Capability Provider
+1. Making requests to an Actor
 
-On top of [Hugo global configuration](https://gohugo.io/overview/configuration/), **Dot** lets you define the following parameters in your `config.toml` (here, values are default).
+### Prerequisites
+In order to follow this guide, you'll need a few things:
+- `wash` (installation covered on [the previous page](../installation))
+- [nats-server](https://docs.nats.io/nats-server/installation) **OR** [docker](https://docs.docker.com/engine/install/), recommended with [docker-compose](https://docs.docker.com/compose/install/)
 
-Note that some of these parameters are explained in details in other sections of this documentation.
+### Starting NATS
+[NATS](https://nats.io/) is a reliable cross-environment messaging system that wasmCloud uses extensively to send messages between the core program components. wasmCloud _can_ be run without using NATS, however this is only recommended in extremely constrained environments, or when NATS is unavailable.

```

Comment:
This might be a bit overwhelming at this stage of the developer journey (since we're assuming they haven't read any reference guides yet). Maybe "NATS is a message broker used by wasmCloud's self-managing **lattice** network" and we leave  it at that (and possibly make the word lattice a link to the lattice reference).

Reply:
Perfect, it felt like a place where I needed to say more than "just start this binary for no reason" and less than exactly what NATS is used for.